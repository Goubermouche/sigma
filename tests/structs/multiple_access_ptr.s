i32 main() {
	struct key {
		i32* value;
	};

	struct user {
		key k;
	};

	user my_user;

	my_user.k.value = cast<i32*>(malloc(sizeof(i32) * 2));
	my_user.k.value[0] = 123;
	my_user.k.value[1] = 321;

	printf("key: %d %d\n", my_user.k.value[0], my_user.k.value[1]);
	ret 0;
}
