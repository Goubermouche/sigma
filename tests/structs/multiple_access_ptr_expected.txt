key: 123 321
